code/game/machinery/Sleeper.dm

```diff
@@ -280,84 +280,83 @@
 	return
 
 
-/obj/machinery/sleeper/attackby(var/obj/item/G as obj, var/mob/user as mob, params)
-	if(istype(G, /obj/item/reagent_containers/glass))
+/obj/machinery/sleeper/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/reagent_containers/glass))
 		if(!beaker)
 			if(!user.drop_item())
-				to_chat(user, "<span class='warning'>\The [G] is stuck to you!</span>")
+				to_chat(user, "<span class='warning'>[I] is stuck to you!</span>")
 				return
 
-			beaker = G
-			G.forceMove(src)
-			user.visible_message("[user] adds \a [G] to \the [src]!", "You add \a [G] to \the [src]!")
+			beaker = I
+			I.forceMove(src)
+			user.visible_message("[user] adds \a [I] to [src]!", "You add \a [I] to [src]!")
 			return
 
 		else
 			to_chat(user, "<span class='warning'>The sleeper has a beaker already.</span>")
 			return
 
-	if(istype(G, /obj/item/screwdriver))
-		if(src.occupant)
+	if(isscrewdriver(I))
+		if(occupant)
 			to_chat(user, "<span class='notice'>The maintenance panel is locked.</span>")
 			return
-		default_deconstruction_screwdriver(user, "[base_icon]-o", "[base_icon]-open", G)
+		default_deconstruction_screwdriver(user, "[base_icon]-o", "[base_icon]-open", I)
 		return
 
-	if(istype(G, /obj/item/wrench))
-		if(src.occupant)
+	if(iswrench(I))
+		if(occupant)
 			to_chat(user, "<span class='notice'>The scanner is occupied.</span>")
 			return
 		if(panel_open)
 			to_chat(user, "<span class='notice'>Close the maintenance panel first.</span>")
 			return
 		if(dir == 4)
 			orient = "LEFT"
-			dir = 8
+			setDir(8)

```

Comment:
Don't we have direction defines?
`1: NORTH`
`2: SOUTH`
`4: EAST`
`8: WEST`

Reply:
Yeah, we do, I just can never remember what they are, because I don't believe they're listed anywhere (they're native BYOND defines).